src/jdk.incubator.vector/share/classes/jdk/incubator/vector/ShortVector.java

```diff
@@ -1808,6 +1810,9 @@ static short reverse(short a) {
         b = (short) (((b & 0x0F0F) << 4) | ((b & 0xF0F0) >>> 4));
         return b;
     }
+    static int reverseBytes(short a) {
+        return (short) ((a << 8) | (a >>> 8));
+    }
```

Comment:
May I ask why not using the built-in Short.reverseBytes[1] here?

[1] https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/lang/Short.java#L565